RELEASE.md

```diff
@@ -0,0 +1,36 @@
+# Releases
+
+## Schedule
+
+The release schedule for this project is ad-hoc. Given the pre-1.0 status of the project we do not have a fixed release cadence. However if a vulnerability is discovered we will respond in accordance with our [security policy](https://github.com/cert-manager/community/blob/main/SECURITY.md) and this response may include a release.
+
+## Process
+
+There is a semi-automated release process for this project. When you create a Git tag with a tagname that has a `v` prefix and push it to GitHub it will trigger the [release workflow].
+
+The release process for this repo is documented below:
+
+1. Create a tag for the new release:
+    ```sh
+   export VERSION=v0.5.0-alpha.0
+   git tag --annotate --message="Release ${VERSION}" "${VERSION}"
+   git push origin "${VERSION}"
+   ```
+2. A GitHub action will see the new tag and do the following:
+    - Build the binaries and create shasums
+    - Create a draft GitHub release
+    - Upload binaries and shasums
```

Comment:
NOTE: The shasum is also signed: 
![image](https://github.com/cert-manager/cmctl/assets/42113979/a10089d5-701b-4a93-b1fd-2cffa6ea5ad1)